upup/models/cloudup/resources/addons/dns-controller.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -38,6 +38,13 @@ spec:
 {{ range $arg := DnsControllerArgv }}
         - "{{ $arg }}"
 {{ end }}
+{{- if .EgressProxy }}
```

Comment:
I feel like there was some back and forth on what the check should be, not sure there is any particular reason I ended up with this one.